### DISCUSSION TITLE
Github extension deprecated

### DISCUSSION THREAD

**Discussion by Izharhussain2022** [AammarTufail/python_ka_chilla2023#3] [General]
@AammarTufail i have installed rest of extensions except github. It says "The extension is deprecated" as shown below:

![image](https://user-images.githubusercontent.com/121819549/210267599-6f99e085-be8f-4104-bbeb-0b52a8eb1d4a.png)

Although we have Githib Copilot. **Does it serve the same purpose?**

![image](https://user-images.githubusercontent.com/121819549/210267763-eb406bbf-08b6-4bab-91d8-2e98dfae22d4.png)

Thanks,

**Comment by nouman073** [comment]
> @AammarTufail i have installed rest of extensions except github. It says "The extension is deprecated" as shown below:
> 
> ![image](https://user-images.githubusercontent.com/121819549/210267599-6f99e085-be8f-4104-bbeb-0b52a8eb1d4a.png)
> 
> Although we have Githib Copilot. **Does it serve the same purpose?**
> 
> ![image](https://user-images.githubusercontent.com/121819549/210267763-eb406bbf-08b6-4bab-91d8-2e98dfae22d4.png)
> 
> Thanks,

the same issue with me , click on the setting icon and reinstall

**Comment by mhamza6051** [comment]
Assignment: Top 10 list of VS Code Extension for python
1. Python, 2. Pylance, 3. Python Indent, 4. Indent-rainbow, 5. Jupiter, 6. Todo MD, 7. DVC, 8. GitLens, 9.BlackBox, 10. Markdown for all

Regards:
M. Hamza

**Comment by baberibrar** [comment]
> @AammarTufail i have installed rest of extensions except github. It says "The extension is deprecated" as shown below:
> 
> ![image](https://user-images.githubusercontent.com/121819549/210267599-6f99e085-be8f-4104-bbeb-0b52a8eb1d4a.png)
> 
> Although we have Githib Copilot. **Does it serve the same purpose?**
> 
> ![image](https://user-images.githubusercontent.com/121819549/210267763-eb406bbf-08b6-4bab-91d8-2e98dfae22d4.png)
> 
> Thanks,

This extension is deprecated. Use the (GitHub Pull Requests and Issues) extension instead.
![Screenshot from 2023-01-03 12-49-33](https://user-images.githubusercontent.com/61284156/210317454-b47c5ddf-3b80-4b5b-9271-c7477a2ebd17.png)
